Tools/gdb/libpython.py

```diff
@@ -1076,7 +1080,10 @@ def _f_builtins(self):
         return self._f_special("f_builtins")
 
     def _f_code(self):
-        return self._f_special("f_code", PyCodeObjectPtr.from_pyobject_ptr)
+        return self._f_special("f_executable", PyCodeObjectPtr.from_pyobject_ptr)
```

Comment:
Can we instead use `self._f_executable` here, test the type, and raise an explicit exception (or return `None`?) if it is not a code object?

Reply:
All uses of this function should know that `self._f_executable` or be guarded against it not being.